src/main/resources/application.conf

```diff
@@ -38,6 +38,8 @@ mantis {
 
   network {
     # Ethereum protocol version
+    # Supported versions:
+    # 63, 64 (experimental version which enables usage of messages with checkpointing information. In the future after ETCM-355, ETCM-356, it will be 66 probably)
```

Comment:
(up for discussion) maybe `network.protocol-version` should be something configured in `blockchains.$someNetwork.network.protocol-version`, given it defines in some way part of the consensus. I mean i'm worry about someone configuring `blockchains.$someNetwork.ecip1097-block-number` but not setting up the proper version for network messages.